mtp_cashbook/apps/disbursements/forms.py

```diff
@@ -384,8 +396,14 @@ def clean_remittance_description(self):
 
     def clean(self):
         super().clean()
-        if self.cleaned_data.get('remittance') == 'no':
-            self.cleaned_data['remittance_description'] = ''
+        remittance = self.cleaned_data.get('remittance')
+        if remittance == 'no':
+            self.cleaned_data['remittance_description'] = self['remittance_description'].initial
+        elif (
+            remittance == 'yes' and
+            self.cleaned_data.get('remittance_description') == self['remittance_description'].initial
+            ):
```

Comment:
almost! i didn't properly notice the indentation in the last review, sorry. the `)` should be at the same indentation level as the `if` and `elif`, so outdented once